### DISCUSSION TITLE
pyscript.web Element.remove()

### DISCUSSION THREAD

**Discussion by obust** [pyscript/pyscript#2149] [Proposals]
analogous to https://developer.mozilla.org/en-US/docs/Web/API/Element/remove

This is useful to fully manipulate the DOM.

Here is an example to empty a list:

```python
# ideal
for il in page["ul"]:
    il.remove()

# today
for il in page["ul"]:
    il._dom_element.remove()
```

**Comment by WebReflection** [comment]
@mchilvers I agree `.remove()` among `.append(...nodes)` and `.prepend(...nodes)`  would be nice additions to our API
